scripts/prometheus/configuration/grafana/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -xe
+
+DASHBOARD_IDS="19646 19647 19679"
+DASHBOARD_PROVISIONNING=/var/tmp/dashboards
+
+download_dashboards() {
+    if [ ! -d $DASHBOARD_PROVISIONNING ]; then
+        mkdir ${DASHBOARD_PROVISIONNING}
+    fi
+
+    for DASHBOARD_ID in $DASHBOARD_IDS;
+    do
+        curl --silent \
+            --insecure \
```

Comment:
Do you know the reason we have to use `--insecure` here?

It seems `grafana/grafana:10.0.1` image has correct TLS certificates already (at least for `grafana.com`):
```
$ docker run -ti --entrypoint curl grafana/grafana:10.0.1 "-I" "https://grafana.com/" 

HTTP/2 200 
server: nginx
date: Mon, 09 Oct 2023 08:25:27 GMT
[...]
```

Reply:
It's mostly a backport from [Grafana official Helm chart](https://github.com/grafana/helm-charts/blob/grafana-6.60.4/charts/grafana/templates/configmap.yaml#L92), but you right this is not interesting because the DigiCert root certificate in the docker image expires in 2031. I removed it.